app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -174,27 +224,31 @@ class Phase(val phaseType: PhaseType) {
         fun getLocalPhases() : List<Phase> {
             return listOf(
                     Phase(PhaseType.LEARN),
-                    Phase(PhaseType.DRAFT),
-                    Phase(PhaseType.COMMUNITY_CHECK),
-                    Phase(PhaseType.CONSULTANT_CHECK),
-                    Phase(PhaseType.DRAMATIZATION),
-                    Phase(PhaseType.CREATE),
+                    Phase(PhaseType.TRANSLATE_REVISE),
+                    Phase(PhaseType.COMMUNITY_WORK),
+                    Phase(PhaseType.ACCURACY_CHECK),
+                    Phase(PhaseType.VOICE_STUDIO),
+                    Phase(PhaseType.FINALIZE),
                     Phase(PhaseType.SHARE))
         }
 
         fun getRemotePhases() : List<Phase> {
             return listOf(
                     Phase(PhaseType.LEARN),
-                    Phase(PhaseType.DRAFT),
-                    Phase(PhaseType.COMMUNITY_CHECK),
+                    Phase(PhaseType.TRANSLATE_REVISE),
+                    Phase(PhaseType.COMMUNITY_WORK),
                     Phase(PhaseType.WHOLE_STORY),
-                    Phase(PhaseType.BACKT),
+                    Phase(PhaseType.BACK_T),
                     Phase(PhaseType.REMOTE_CHECK),
-                    Phase(PhaseType.DRAMATIZATION),
-                    Phase(PhaseType.CREATE),
+                    Phase(PhaseType.VOICE_STUDIO),
+                    Phase(PhaseType.FINALIZE),
                     Phase(PhaseType.SHARE))
         }
 
+        /**
+         * get the filename for the HTML help doc
+         * @return String

```

Comment:
\* also, when refactoring we need to think through how localized help files are (re)named.

Reply:
This is taken care of.